Docker.README.md

```diff
@@ -0,0 +1,48 @@
+# Running duke-data-service in docker
```

Comment:
These were just my notes for getting to the point of better integration testing.
I just wanted to keep them for when I get around to those changes.